components/textlocal/sources/new-sent-api-message/new-sent-api-message.js

```diff
@@ -0,0 +1,66 @@
+const common = require("../common/timer-based");
+
+module.exports = {
+...common,
+  key: "textlocal-new-sent-api-message",
+  name: "New Sent API Message",
+  version: "0.0.2",
```

Comment:
Has this component been published publicly before?

Reply:
No, this is my bad. Since I created a PR for this particular component and this was a follow-up PR, I bumped the version. But that PR was never merged to `master` so this should be `0.0.1`. I will fix it.